### ISSUE TITLE
Auto start - error creating startup entry Ubuntu 16.04

### ISSUE THREAD

**Issue by klabarge** [qzind/tray#131]
**OS:** Ubuntu 16.04
**Java:** Oracle 8u111
**QZ Tray:** 2.0.2

When trying to check "Automatically start", QZ Tray throws an error: "Error creating startup entry"

This is the error that shows in the terminal:
```
[DEBUG] 2016-11-13 22:06:58,456 @ qz.utils.ShellUtilities:?
	Executing: [cp, /usr/share/applications/qz-tray.desktop, /home/kyle/.config/autostart/]
```

**Comment by tresf** [comment]
@klabarge can you please provide the output of the following command:


```bash
cp /usr/share/applications/qz-tray.desktop /home/kyle/.config/autostart/
```

**Comment by klabarge** [comment]
> can you please provide the output of the following command:

```
kyle@lagunitas:~$ cp /usr/share/applications/qz-tray.desktop /home/kyle/.config/autostart/cp: cannot create regular file '/home/kyle/.config/autostart/': Not a directory
kyle@lagunitas:~$ 
```

It looks like `autostart` is not a directory.

```
kyle@lagunitas:~$ ls /home/kyle/.config/
compiz-1/         gtk-3.0/          pulse/            user-dirs.dirs
dconf/            ibus/             unity/            user-dirs.locale
evolution/        libaccounts-glib/ update-notifier/  
gnome-session/    nautilus/         upstart/          
kyle@lagunitas:~$ ls /home/kyle/.config/
```

**Comment by tresf** [comment]
Kyle can you try adding this workaround to[ `LinuxDeploy.java'](https://github.com/qzind/tray/blob/2.0/src/qz/deploy/LinuxDeploy.java)

```diff
@Override
    public boolean createStartupShortcut() {
+        ShellUtilities.execute(new String[] {
+                "mkdir", "-p", System.getProperty("user.home") + "/.config/autostart/"
+        });
        return createShortcut(System.getProperty("user.home") + "/.config/autostart/");
    }
```

**Comment by klabarge** [comment]
Closed via f65e527b66110f2f9f6766cb6238f0ca959aec9c
